script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -68,14 +68,14 @@ I'm speechless.
 Sometimes I wonder why I'm his best friend.
 <Page18>
 Orange dyed hair and pierced ears.
-He wears an anti-social attire that gives a bad image as if he were ready to fight with anyone at any time.
+He wears rebellious attire that gives off a bad image as if he's ready to fight with anyone at any time.
```

Comment:
"as if he's ready" feels a bit strange to me, I'd say "as though he was ready", but maybe I'm wrong.